demo/offline_section.js

```diff
@@ -201,7 +210,7 @@ shakaDemo.refreshAssetList_ = function() {
     group.removeChild(group.firstChild);
   }
 
-  shakaDemo.setupOfflineAssets_();
+  return shakaDemo.setupOfflineAssets_();
```

Comment:
Since this returns a value now, please update the annotations with @return {!Promise}

Reply:
Done in https://github.com/google/shaka-player/pull/1001/commits/e3af7f17674d23d3a8772a5c1e828b617bd81b81